apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_actions.erl

```diff
@@ -111,11 +111,11 @@
                        zh => <<"桥接挂载点"/utf8>>},
             description => #{
                 en => <<"MountPoint for bridge topic:<br/>"
-                        "Example: The topic of messages sent to `topic1` on local node "
-                        "will be transformed to `bridge/aws/${node}/topic1`">>,
+                        "Example: The topic of messages sent to <code>topic1</code> on local node "
```

Comment:
are you sure this is going to work in 4.4 dashboard ?

Reply:
I've tried it on both open-source and enterprise version, no problems with the display of the strings (bridge/aws/${node}/topic1), the highlight effect is not that obvious though:

<img width="495" alt="image" src="https://user-images.githubusercontent.com/11696433/217460926-d0c9fd6e-183f-4c25-8c61-e456fcad1fff.png">

BTW, the `<br/>` works as expected.